### PR TITLE
Support selectpicker type of Select

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/automate/test_provisioning_dialogs.py cfme/tests/automate/test_service_dialog.py cfme/tests/automate/test_customization_paginator.py -v --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 28bbc25d03333063485a26e9a250439f451835e1:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f97419b94c8c0daa8e84563660bcb1ba26681cbd:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by mfalesni** [comment]
So just tested it in automation on my laptop on an upstream appliance and it seems to work properly. @psav @seandst what do you say on this modification?

File: cfme/fixtures/pytest_selenium.py [hunk 1]
```diff
@@ -1237,14 +1240,29 @@ def none(self):
...
+            var opt_elements = arguments[0].options;
+            for(var i = 0; i < opt_elements.length; i++){
+                var option = opt_elements[i];
+                result_arr.push([option.innerHTML, option.getAttribute("value")]);
+            }
+            return result_arr;
+        """)
+        options = execute_script(script, element(self))
+        parser = HTMLParser()
+        return [self.Option(parser.unescape(option[0]), option[1]) for option in options]
```
Comment (psav) [inline cfme/fixtures/pytest_selenium.py]: Is this unescape going to alter anything we already have that uses escaped strings in matching?

Comment (mfalesni) [inline cfme/fixtures/pytest_selenium.py]: Nope. Selenium apparently does that too (the `<None>` things and so, pure javascript yields `&lt;` and similar things). Using javascript, I can read the select always no matter in what position/visibility it is.

**Pull request merged**